ipn/ipnstate/ipnstate.go

```diff
@@ -701,10 +701,15 @@ type PingResult struct {
 	Err            string
 	LatencySeconds float64
 
-	// Endpoint is the ip:port if direct UDP was used.
-	// It is not currently set for TSMP pings.
+	// Endpoint is a string of the form "{ip}:{port}" if direct UDP was used. It
+	// is not currently set for any other ping type (TSMP, peer relay, etc).
 	Endpoint string
 
+	// PeerRelay is a string of the form "{ip}:{port}:vni:{vni}" if a peer
+	// relay was used. It is not currently set for any other ping type (TSMP,
+	// direct, etc).
+	PeerRelay string `json:",omitempty"`
```

Comment:
It's interesting that we json omitempty `Endpoint` in `tailcfg.PingResponse`, but we don't do the same in this struct.

I don't believe there's a special meaning for a zero value vs omitted upon decode `Endpoint` or `PeerRelay` here, so it shouldn't hurt either way.